cmd/cli/main.go

```diff
@@ -19,7 +19,7 @@ func main() {
 
 	// Root
 	var rootCmd = &cobra.Command{
-		Use: "bashflux",
+		Use: "mainflux-cli",
```

Comment:
Maybe we can pick some other name for this? It will be easier to remember and use if we take shorter and simpler name. Simply `mainflux` (since it's obvious that it's cli tool)?

Reply:
Actually, Makefile currently adds `mainflux-` prefix, and it's easy to add `cli` to current buils system like that. This `mainflux-` prefix is added because binaries are installed in `$GOBIN`. Let's keep it like this for now, and we'll see later for the refinement.